devfile.yaml

```diff
@@ -11,13 +11,13 @@
 #
 schemaVersion: 2.3.0
 attributes:
-  controller.devfile.io/storage-type: ephemeral
+  controller.devfile.io/storage-type: per-workspace
 metadata:
   name: che-operator
 components:
   - name: tooling-container
     container:
-      image: quay.io/eclipse/che-operator-dev:latest
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
```

Comment:
Please, revert this line.
dev image is based on quay.io/devfile/universal-developer-image:ubi8-latest

Reply:
Done, I removed the associated commit. However, the reason why I changed the image being used is because the che-operator-dev container image is outdated and needs to be rebuilt to pull in the latest changes from the UDI.